app/views/embeddable/multiple_choice_answers/_lightweight.html.haml

```diff
@@ -12,13 +12,12 @@
       - if embeddable.show_as_menu
         = f.select :answers, options_from_collection_for_select(embeddable.choices, 'id', 'choice', (embeddable.answers.empty? ? nil : embeddable.answers.last.id) ), { :include_blank => t("PICK_ONE")}, { :data => {:button_id => check_button_string} }
       - else
-        - embeddable.choices.each do |choice|
-          %label{ :for => "embeddable_multiple_choice_answer_answers_#{choice.id}" }
-            - if embeddable.multi_answer
-              = check_box_tag 'embeddable_multiple_choice_answer[answers][]', choice.id, embeddable.answers.include?(choice), :id => "embeddable_multiple_choice_answer_answers_#{choice.id}", :data => {:button_id => check_button_string}
+        .choice-container{:style => embeddable.is_likert ? "display:table; width:100%;" : ""}
```

Comment:
@sagartalla the layout type should be added as a class to the .choice-container then the inline style info here can be moved to a css file. As I mentioned in another comment, this can then also be used for the horizontal and vertical styling.